Failure cluster [28311165...] Make integration test TestQuotaLimitService wait longer, to avoid flakes

### Failure cluster [28311165688ff5ded2f4](https://go.k8s.io/triage#28311165688ff5ded2f4)

##### Error text:
```
=== RUN   TestQuotaLimitService
I1230 00:54:57.057578  122720 controller.go:163] quota evaluator worker shutdown
I1230 00:54:57.057590  122720 controller.go:163] quota evaluator worker shutdown
I1230 00:54:57.057591  122720 controller.go:163] quota evaluator worker shutdown
I1230 00:54:57.057599  122720 controller.go:163] quota evaluator worker shutdown
I1230 00:54:57.057603  122720 controller.go:163] quota evaluator worker shutdown
W1230 00:54:57.058001  122720 mutation_detector.go:53] Mutation detector is enabled, this will result in memory leakage.
I1230 00:54:57.058561  122720 apf_controller.go:218] NewTestableController "Controller" with serverConcurrencyLimit=600, requestWaitLimit=15s, name=Controller, asFieldManager="api-priority-and-fairness-config-consumer-v1"
I1230 00:54:57.058634  122720 apf_controller.go:769] No exempt PriorityLevelConfiguration found, imagining one
I1230 00:54:57.058647  122720 apf_controller.go:769] No catch-all PriorityLevelConfiguration found, imagining one
W1230 00:54:57.058679  122720 mutation_detector.go:53] Mutation detector is enabled, this will result in memory leakage.
W1230 00:54:57.058700  122720 mutation_detector.go:53] Mutation detector is enabled, this will result in memory leakage.
I1230 00:54:57.058748  122720 services.go:51] Setting service IP to "10.0.0.1" (read-write).
I1230 00:54:57.058768  122720 instance.go:318] Node port range unspecified. Defaulting to 30000-32767.
I1230 00:54:57.058782  122720 instance.go:274] Using reco
```
#### Recent failures:
[2021/12/30 上午8:28:34 ci-kubernetes-integration-master](https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-integration-master/1476348992219516928)


/kind failing-test
<!-- If this is a flake, please add: /kind flake -->
/kind flake
<!-- Please assign a SIG using: /sig SIG-NAME -->
/sig api-machinery

`    quota_test.go:214: resource services, expected 2, actual 0
    quota_test.go:221: error waiting or ResourceQuota status: timed out waiting for the condition
W1230 00:55:11.755545  122720 apf_controller.go:831] no match found for request &request.RequestInfo{IsResourceRequest:true, Path:"/api/v1/namespaces/quota/services", Verb:"create", APIPrefix:"api", APIGroup:"", APIVersion:"v1", Namespace:"quota", Resource:"services", Subresource:"", Name:"", Parts:[]string{"services"}} and user &user.DefaultInfo{Name:"", UID:"", Groups:[]string(nil), Extra:map[string][]string(nil)}; selecting catchAll={"metadata":{"name":"catch-all","uid":"19e64606-bc43-4f76-9dff-08827c0429b0","resourceVersion":"58067","generation":1,"creationTimestamp":"2021-12-30T00:54:57Z","annotations":{"apf.kubernetes.io/autoupdate-spec":"true"}},"spec":{"priorityLevelConfiguration":{"name":"catch-all"},"matchingPrecedence":10000,"distinguisherMethod":{"type":"ByUser"},"rules":[{"subjects":[{"kind":"Group","group":{"name":"system:unauthenticated"}},{"kind":"Group","group":{"name":"system:authenticated"}}],"resourceRules":[{"verbs":["*"],"apiGroups":["*"],"resources":["*"],"clusterScope":true,"namespaces":["*"]}],"nonResourceRules":[{"verbs":["*"],"nonResourceURLs":["*"]}]}]},"status":{"conditions":[{"type":"Dangling","status":"False","lastTransitionTime":"2021-12-30T00:54:57Z","reason":"Found","message":"This FlowSchema references the PriorityLevelConfiguration object named \"catch-all\" and it exists"}]}} as fallback flow schema
W1230 00:55:11.773754  122720 apf_controller.go:831] no match found for request &request.RequestInfo{IsResourceRequest:true, Path:"/api/v1/namespaces/quota/resourcequotas/quota/status", Verb:"update", APIPrefix:"api", APIGroup:"", APIVersion:"v1", Namespace:"quota", Resource:"resourcequotas", Subresource:"status", Name:"quota", Parts:[]string{"resourcequotas", "quota", "status"}} and user &user.DefaultInfo{Name:"", UID:"", Groups:[]string(nil), Extra:map[string][]string(nil)}; selecting catchAll={"metadata":{"name":"catch-all","uid":"19e64606-bc43-4f76-9dff-08827c0429b0","resourceVersion":"58067","generation":1,"creationTimestamp":"2021-12-30T00:54:57Z","annotations":{"apf.kubernetes.io/autoupdate-spec":"true"}},"spec":{"priorityLevelConfiguration":{"name":"catch-all"},"matchingPrecedence":10000,"distinguisherMethod":{"type":"ByUser"},"rules":[{"subjects":[{"kind":"Group","group":{"name":"system:unauthenticated"}},{"kind":"Group","group":{"name":"system:authenticated"}}],"resourceRules":[{"verbs":["*"],"apiGroups":["*"],"resources":["*"],"clusterScope":true,"namespaces":["*"]}],"nonResourceRules":[{"verbs":["*"],"nonResourceURLs":["*"]}]}]},"status":{"conditions":[{"type":"Dangling","status":"False","lastTransitionTime":"2021-12-30T00:54:57Z","reason":"Found","message":"This FlowSchema references the PriorityLevelConfiguration object named \"catch-all\" and it exists"}]}} as fallback flow schema
I1230 00:55:11.778903  122720 httplog.go:129] "HTTP" verb="PUT" URI="/api/v1/namespaces/quota/resourcequotas/quota/status" latency="5.225664ms" userAgent="Go-http-client/1.1" audit-ID="81d67947-d445-4584-80c7-66dcb2621b3f" srcIP="127.0.0.1:48896" apf_pl="catch-all" apf_fs="catch-all" apf_fd="" apf_iseats=1 apf_fseats=0 resp=200
I1230 00:55:11.783592  122720 alloc.go:327] "allocated clusterIPs" service="quota/lb-svc-withnp2" clusterIPs=map[IPv4:10.0.0.114]
I1230 00:55:11.783782  122720 httplog.go:129] "HTTP" verb="POST" URI="/api/v1/namespaces/quota/services" latency="28.29752ms" userAgent="Go-http-client/1.1" audit-ID="54e3d6a3-e720-4d42-90e5-c4eb340170e8" srcIP="127.0.0.1:48884" apf_pl="catch-all" apf_fs="catch-all" apf_fd="" apf_iseats=1 apf_fseats=0 resp=201
skipped 38 lines unfold_more
I1230 00:55:21.826914  122720 httplog.go:129] "HTTP" verb="GET" URI="/api/v1/namespaces/quota/resourcequotas/quota" latency="4.620533ms" userAgent="Go-http-client/1.1" audit-ID="2fde2875-6641-4e68-8ac9-d1b9c5d8156d" srcIP="127.0.0.1:48884" apf_pl="catch-all" apf_fs="catch-all" apf_fd="" apf_iseats=1 apf_fseats=0 resp=200
    quota_test.go:214: resource services, expected 3, actual 2`

polls a ResourceQuota status with timeout 10s may be not enough, we should enlarge it into 20s

--- a/test/integration/auth/auth_test.go
+++ b/test/integration/auth/auth_test.go
@@ -62,6 +62,7 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	v1 "k8s.io/client-go/tools/clientcmd/api/v1"
+	resttransport "k8s.io/client-go/transport"
 	kubeapiservertesting "k8s.io/kubernetes/cmd/kube-apiserver/app/testing"
 	"k8s.io/kubernetes/pkg/apis/autoscaling"
 	api "k8s.io/kubernetes/pkg/apis/core"
@@ -555,11 +556,9 @@ func TestAuthModeAlwaysDeny(t *testing.T) {
 	controlPlaneConfig.GenericConfig.Authorization.Authorizer = authorizerfactory.NewAlwaysDenyAuthorizer()
 	_, s, closeFn := framework.RunAnAPIServer(controlPlaneConfig)
 	defer closeFn()
-
 	ns := framework.CreateTestingNamespace("auth-always-deny", s, t)
 	defer framework.DeleteTestingNamespace(ns, s, t)
-
-	transport := http.DefaultTransport
+	transport := resttransport.NewBearerAuthRoundTripper(framework.UnprivilegedUserToken, http.DefaultTransport)
 
 	for _, r := range getTestRequests(ns.Name) {
 		bodyBytes := bytes.NewReader([]byte(r.body))

--- a/test/integration/controlplane/synthetic_controlplane_test.go
+++ b/test/integration/controlplane/synthetic_controlplane_test.go
@@ -38,8 +38,10 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	authauthenticator "k8s.io/apiserver/pkg/authentication/authenticator"
 	"k8s.io/apiserver/pkg/authentication/group"
 	"k8s.io/apiserver/pkg/authentication/request/bearertoken"
+	authenticatorunion "k8s.io/apiserver/pkg/authentication/request/union"
 	"k8s.io/apiserver/pkg/authentication/user"
 	"k8s.io/apiserver/pkg/authorization/authorizer"
 	"k8s.io/apiserver/pkg/authorization/authorizerfactory"
@@ -141,6 +143,15 @@ func TestEmptyList(t *testing.T) {
 
 func initStatusForbiddenControlPlaneConfig() *controlplane.Config {
 	controlPlaneConfig := framework.NewIntegrationTestControlPlaneConfig()
+	controlPlaneConfig.GenericConfig.Authentication.Authenticator = authenticatorunion.New(
+		authauthenticator.RequestFunc(func(req *http.Request) (*authauthenticator.Response, bool, error) {
+			return &authauthenticator.Response{
+				User: &user.DefaultInfo{
+					Name:   "unprivileged",
+					Groups: []string{user.AllAuthenticated},
+				},
+			}, true, nil
+		}))
 	controlPlaneConfig.GenericConfig.Authorization.Authorizer = authorizerfactory.NewAlwaysDenyAuthorizer()
 	return controlPlaneConfig
 }
@@ -178,7 +189,7 @@ func TestStatus(t *testing.T) {
 			statusCode:         http.StatusForbidden,
 			reqPath:            "/apis",
 			reason:             "Forbidden",
-			message:            `forbidden: User "" cannot get path "/apis": Everything is forbidden.`,
+			message:            `forbidden: User "unprivileged" cannot get path "/apis": Everything is forbidden.`,
 		},
 		{
 			name:               "401",

--- a/test/integration/framework/controlplane_utils.go
+++ b/test/integration/framework/controlplane_utils.go
@@ -61,6 +61,10 @@ import (
 	netutils "k8s.io/utils/net"
 )
 
+const (
+	UnprivilegedUserToken = "unprivileged-user"
+)
+
 // Config is a struct of configuration directives for NewControlPlaneComponents.
 type Config struct {
 	// If nil, a default is used, partially filled configs will not get populated.
@@ -80,11 +84,16 @@ func (alwaysAllow) Authorize(ctx context.Context, requestAttributes authorizer.A
 	return authorizer.DecisionAllow, "always allow", nil
 }
 
-// alwaysEmpty simulates "no authentication" for old tests
-func alwaysEmpty(req *http.Request) (*authauthenticator.Response, bool, error) {
+// unsecuredUser simulates requests to the unsecured endpoint for old tests
+func unsecuredUser(req *http.Request) (*authauthenticator.Response, bool, error) {
+	auth := req.Header.Get("Authorization")
+	if len(auth) != 0 {
+		return nil, false, nil
+	}
 	return &authauthenticator.Response{
 		User: &user.DefaultInfo{
-			Name: "",
+			Name:   "system:unsecured",
+			Groups: []string{user.SystemPrivilegedGroup, user.AllAuthenticated},
 		},
 	}, true, nil
 }
@@ -171,12 +180,17 @@ func startAPIServerOrDie(controlPlaneConfig *controlplane.Config, incomingServer
 	tokens[privilegedLoopbackToken] = &user.DefaultInfo{
 		Name:   user.APIServerUser,
 		UID:    uuid.New().String(),
-		Groups: []string{user.SystemPrivilegedGroup},
+		Groups: []string{user.SystemPrivilegedGroup, user.AllAuthenticated},
+	}
+	tokens[UnprivilegedUserToken] = &user.DefaultInfo{
+		Name:   "unprivileged",
+		UID:    uuid.New().String(),
+		Groups: []string{user.AllAuthenticated},
 	}
 
 	tokenAuthenticator := authenticatorfactory.NewFromTokens(tokens, controlPlaneConfig.GenericConfig.Authentication.APIAudiences)
 	if controlPlaneConfig.GenericConfig.Authentication.Authenticator == nil {
-		controlPlaneConfig.GenericConfig.Authentication.Authenticator = authenticatorunion.New(tokenAuthenticator, authauthenticator.RequestFunc(alwaysEmpty))
+		controlPlaneConfig.GenericConfig.Authentication.Authenticator = authenticatorunion.New(tokenAuthenticator, authauthenticator.RequestFunc(unsecuredUser))
 	} else {
 		controlPlaneConfig.GenericConfig.Authentication.Authenticator = authenticatorunion.New(tokenAuthenticator, controlPlaneConfig.GenericConfig.Authentication.Authenticator)
 	}

